tests/lib/rules/no-magic-numbers.js

```diff
@@ -213,6 +213,26 @@ ruleTester.run("no-magic-numbers", rule, {
             code: "f(-100n)",
             options: [{ ignore: ["-100n"] }],
             parserOptions: { ecmaVersion: 2020 }
+        },
+        {
+            code: "const { param = 123 } = sourceObject;",
+            options: [{ ignoreDefaultValues: true }],
+            env: { es6: true }
+        },
+        {
+            code: "const func = ({ param = 123 }) => {}",
```

Comment:
This test covers another variant of object destructuring similar to the prior case, but I noticed these tests haven't been updated to include cases for default function parameters since we added ES6 support. While you're working on this, would you mind adding a valid and invalid case with the new option enabled and disabled, respectively, for `function func(param = 123) {}`? (Optionally a function expression and an arrow function if you want to be extra thorough, though the handling _should_ be the same.)

Reply:
Not 100% sure I got this right. Could you check the last commit?

Reply:
Those two cases work, thank you!